HCCGo/app/js/jobSubmissionCtrl.js

```diff
@@ -146,7 +146,7 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
 
     // Separate SBATCH options from commands
     job.commands = editor.getValue();
-    var other = editor.getValue().split("\n");
+    var other = editor.getValue().split(/\r?\n/);
```

Comment:
Yeah, sounds good to me.  If it successfully submitted after having `\r`'s in the file, then it removed them.  Natasha wasn't even able to submit a job with `\r`.